app/models/perennial_produce.rb

```diff
@@ -1,6 +1,5 @@
 class PerennialProduce < Produce
```

Comment:
Wasn't totally sure what to keep with the validations here; could you look this over and make sure I made the correct changes?